themes/bootstrap3/templates/Recommend/WebResults.phtml

```diff
@@ -4,7 +4,7 @@
 ?>
 <?php if (!empty($results)): ?>
 <div class="sidegroup">
-  <h4><?=$this->transEsc('Library Web Search')?></h4>
+  <h2><?=$this->transEsc('Library Web Search')?></h2>
```

Comment:
I can't test Web search

Reply:
Before:

![image](https://github.com/vufind-org/vufind/assets/309069/8eaf64fe-ce8e-4f1c-bdf2-8903963a1713)

After:

![image](https://github.com/vufind-org/vufind/assets/309069/643d16c5-321d-4009-b6c0-f530f528eba4)

... looks fine to me!